protocol/agents.json

```diff
@@ -232,6 +232,8 @@
     "php-laravel": "ably-php/${version} laravel",
     "python": "ably-python/${version}",
     "ruby": "ably-ruby/${version}",
-    "ruby-rest": "ably-ruby/${version} ably-ruby-rest"
+    "ruby-rest": "ably-ruby/${version} ably-ruby-rest",
+    "flutter": "ably-flutter/${version}",
+    "kafka-connect": "kafka-connect-ably/${version}"
```

Comment:
As described [this section on ablyLibMappings](https://github.com/ably/ably-common/tree/main/protocol#ablylibmappings), only libraries which continue to send the deprecated `X-Ably-Lib` header should be added to this list, but I don't believe either flutter or kafka connect do so?

Reply:
My bad. You are completely right